behave/features/cla_public/public_eligible.feature

```diff
@@ -73,7 +73,7 @@ Scenario: Complete in scope housing check
   And I click continue
   And I am taken to the "Review your answers" page located on "/review"
   And I click Confirm
```

Comment:
I think you need to add a "And I click Confirm" step here. 
This test consistently fails because it cannot get past this page:
![_121459_29_04_2024](https://github.com/ministryofjustice/cla-end-to-end-tests/assets/143531642/b6294cec-3080-40ef-9cf4-f5bfca4e3536)